nilearn/decoding/searchlight.py

```diff
@@ -319,6 +355,12 @@ def fit(self, imgs, y, groups=None):
         process_mask, process_mask_affine = masking.load_mask_img(
             process_mask_img
         )
+        if process_mask.shape != imgs.shape[:3]:
+            raise ValueError(
+                "The mask image and the 4D input images must"
+                " have matching dimensions."
+            )
```

Comment:
Given that we use `coord_transform` right after: I don't think we actually want to check for this.

If the images have different dimensions, then `coord_transform` will take care of that.

@bthirion can you confirm?

Reply:
Were you able to check that this condition is not necessary ?